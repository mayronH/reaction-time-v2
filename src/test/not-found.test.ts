import { test, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import NotFound from '../views/404View.vue'

test('not found view', async () => {
  expect(NotFound).toBeTruthy()

  const wrapper = mount(NotFound)

  expect(wrapper.html()).toMatchSnapshot()
  expect(wrapper.text()).toContain('Page not Found')
})
