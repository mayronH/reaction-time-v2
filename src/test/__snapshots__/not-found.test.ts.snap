// Vitest Snapshot v1

exports[`not found view 1`] = `
"<main class=\\"content\\" data-v-ac76ae68=\\"\\">
  <h1 data-v-ac76ae68=\\"\\">404</h1>
  <h2 data-v-ac76ae68=\\"\\">Page not Found</h2>
</main>"
`;
